src/libraries/System.Runtime/tests/System/Reflection/InvokeEmit/System.Runtime.ReflectionInvokeEmit.Tests.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TestRuntime>true</TestRuntime>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
```

Comment:
Why do we need separate builds for Windows and Linux?

Reply:
I'll let this CI run finish, and then change it to just `NetCoreAppCurrent`. I suspect that other tests were failing in the original project so we should be OK to change it here for just the subset of the reflection tests.